Roomz/src/main/webapp/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
```

Comment:
can you include a screenshot of the current page in the PR description? same for future PRs, so that it's easy to keep track of the visual progress with each change

Reply:
Sure, Alan! I will edit the description and add the screenshot! Thank you!

Reply:
Hi Alan, thanks for your feedback! I have updated the PR title and description to include setting up the skeleton of the web server.

Reply:
Thanks!